### PR TITLE
Document custom connectors that require additional jars in presto-admin

### PR DESCRIPTION
**Pull request by mattsfuller**
Document an example in the presto-admin-commands documentation to add a jar and connector by using two presto-admin commands.

### CODE REVIEW

**Comment by mattsfuller** [comment]
@rschlussel Can you please review?

File: docs/presto-admin-commands.rst [hunk 1]
```diff
@@ -331,6 +331,29 @@ once, you could run ::
     sudo ./presto-admin connector add
     sudo ./presto-admin server restart
 
+Adding a Custom Connector
+--------------------------
+Not all connectors are available in Presto by default. For non default
```
Comment (cawallin) [inline docs/presto-admin-commands.rst]: Just say "In order to add a custom connector not included with Presto, the jar must be ..."

File: docs/presto-admin-commands.rst [hunk 2]
```diff
@@ -331,6 +331,29 @@ once, you could run ::
...
+   sudo ./presto-admin connector add my_connector
+   sudo ./presto-admin server restart
+
+The ``add_jar`` command assumes the default plugin location of
+``/usr/lib/presto/lib/plugin`` (see `plugin add_jar`_).  As with the
+default connectors, a ``my_connector.properties`` file must be
+created. Refer to the custom connector's documentation for the
+properties to specify.
+
+The ``plugin add_jar`` command can be run multiple times for when the
```
Comment (cawallin) [inline docs/presto-admin-commands.rst]: Just mention that it works with either jars or directories

File: docs/presto-admin-commands.rst [hunk 3]
```diff
@@ -331,6 +331,25 @@ once, you could run ::
     sudo ./presto-admin connector add
     sudo ./presto-admin server restart
 
+Adding a Custom Connector
+--------------------------
+In order to install a custom connector not included with Presto, the jar must be
+added to the Presto plugin location using the ``plugin add_jar`` command before
```
Comment (rschlussel) [inline docs/presto-admin-commands.rst]: re: 'before': the order of plugin add_jar and connector add doesn't matter, but this is fine.  They have to be executed in some order

**Comment by rschlussel** [comment]
lgtm

**Pull request closed without merging**